internals/cli/cli_test.go

```diff
@@ -168,6 +169,37 @@ func (s *PebbleSuite) TestGetEnvPaths(c *C) {
 	c.Assert(socketPath, Equals, "/path/to/socket")
 }
 
+func (s *PebbleSuite) TestGetAdditionalAdminUIDs(c *C) {
+	defer os.Unsetenv("PEBBLE_ADMINS")
+
+	os.Setenv("PEBBLE_ADMINS", "")
+	uids, err := cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, HasLen, 0)
+
+	os.Setenv("PEBBLE_ADMINS", "123")
+	uids, err = cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, DeepEquals, []sys.UserID{123})
+
+	os.Setenv("PEBBLE_ADMINS", "123,345")
+	uids, err = cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, DeepEquals, []sys.UserID{123, 345})
+
+	os.Setenv("PEBBLE_ADMINS", "123, 345")
```

Comment:
I think we should probably use `strutil.CommaSeparatedList`, which we use elsewhere in Pebble, to parse this. That does `strings.TrimSpace` on the values, which I think is a bit more user-friendly too. (We'd still have to whip over the list and convert to ints.)

Reply:
Great idea.